src/scales/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+/* eslint-disable global-require */
```

Comment:
I might rewrite this so that we don't need the `eslint-disable`. When we eventually move from `require` to `import` I think we will need to do this anyway, so might as well start structuring things in that format now.
```
var category = require('./scale.category');
var linear = require('./scale.linear');
var logarithmic = require('./scale.logarithmic');
var radialLinear = require('./scale.radialLinear');
var time = require('./scale.time');

module.exports = {
	category: category,
	linear: linear,
	logarithmic: logarithmic,
	radialLinear: radialLinear,
	time: time
};
```

Reply:
Agreed. This comes from my branch pre-rollup and was generating smaller code. Rollup do a better job, both formats generate the same output:

```javascript
var scales = {
	category: scale_category,
	linear: scale_linear,
	logarithmic: scale_logarithmic,
	radialLinear: scale_radialLinear,
	time: scale_time
};

// with ...

var scale_category = core_scale.extend( ... )
```